lib/ansible/modules/packaging/os/yum_repository.py

```diff
@@ -635,7 +638,7 @@ def main():
         argument_spec=dict(
             async=dict(type='bool'),
             bandwidth=dict(),
-            baseurl=dict(),
+            baseurl=dict(type='raw'),
```

Comment:
Doesn't this break the backward compatibility? Previously the type was the default `str`. I guess the `list` will force users to use YAML list instead of string?